x-pack/plugins/reporting/server/usage/get_export_stats.ts

```diff
@@ -127,7 +130,7 @@ export const getExportStats = (
         featureAvailability
       ),
     }),
-    {}
+    {} as JobTypes
```

Comment:
this change, and below in this file, is a minor improvement for stricter typescript